scripts/repository.lic

```diff
@@ -1466,16 +1477,23 @@ elsif (cmd[0] =~ /^show-?updat(?:e|able)$/i)
   Settings['updatable'] ||= Hash.new
   Settings['updatable'][:scripts] ||= Array.new
   Settings['updatable'][:mapdb] ||= Hash.new
-  output = "\n"
-  output.concat "      Lich: #{Settings['updatable'][:lich] ? 'on' : 'off'}\n"
-  output.concat "    map db: #{Settings['updatable'][:mapdb][XMLData.game] ? 'on' : 'off'}\n"
+  table_rows = []
+  table_rows.push([{:value => " Lich: #{Settings['updatable'][:lich] ? 'on' : 'off'}", :colspan => 3}])
+  table_rows.push([{:value => "mapdb: #{Settings['updatable'][:mapdb][XMLData.game] ? 'on' : 'off'}", :colspan => 3}])
+  table_rows.push(:separator)
+  table_rows.push(['Script/File', 'Author', 'Game'])
+  table_rows.push(:separator)
   if Settings['updatable'][:scripts].empty?
-    output.concat "   scripts: (none)\n"
+    table_rows.push([{:value => "No Scripts Updatable", :colspan => 3}])
   else
-    output.concat "   scripts: #{Settings['updatable'][:scripts].collect { |script_updatable| script_updatable[:filename].sub(/\.lic$/, '') }.join(', ')}\n"
+    Settings['updatable'][:scripts].sort_by { |script| script[:filename] }.each { |script_updatable|
+      table_rows.push([script_updatable[:filename], script_updatable[:author], script_updatable[:game]])
+    }
   end
-  output.concat "\n"
-  respond output
+  table = Terminal::Table.new(
+    rows: table_rows
+  )
+  _respond "<output class=\"mono\"/>\n" + table.to_s + "\n<output class=\"\"/>"
```

Comment:
```
table_str = table.to_s
table_str.gsub(/Alastir/, "#{monsterbold_start}Alastir#{monsterbold_end}")
_respond "<output class=\"mono\"/>\n" + table_str + "\n<output class=\"\"/>"
```

Reply:
Uncertain if I agree with monsterbolding a name. Does it mean a good thing or a bad thing to someone unfamiliar to the reasoning of the monsterbold. Also not sure I want to take on the moderation, and never ending battle that comes with it, of an open source system that right now the server for it are not all controlled by the community but by a single server owner (Tillmen).

Reply:
i was (half) kidding. More like 10%.